tests/ssl-opt.sh

```diff
@@ -8478,6 +8487,27 @@ run_test    "export keys functionality" \
             -c "EAP-TLS IV is:" \
             -s "EAP-TLS IV is:"
 
+# TLS1.3 test cases
+# TODO: remove or rewrite this test case if #4832 is resolved.
+requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_2
+requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL
+skip_handshake_stage_check
+run_test    "TLS1.3: Not supported version check: tls1_2 and tls1_3" \
+            "$P_SRV debug_level=4 min_version=tls1_2 max_version=tls1_3" \
+            "$P_CLI debug_level=4 min_version=tls1_2 max_version=tls1_3" \
+            1 \
+            -s "SSL - Invalid value in SSL config" \
```

Comment:
If you opt for the more detailed debug line suggested above, we should grep for it here.